scripts/tests/test_safe_template_linter.py

```diff
@@ -361,6 +371,18 @@ def test_check_mako_expression_display_name(self):
         {
             'expression':
                 textwrap.dedent("""
+                    <%
+                        a_link_start = '<a class="link-courseURL" rel="external" href="'
+                        a_link_end = '">' + _("your course summary page") + '</a>'
+                        a_link = a_link_start + lms_link_for_about_page + a_link_end
+                        text = _("Introductions, prerequisites, FAQs that are used on %s (formatted in HTML)") % a_link
+                    %>
+                """),
+            'rule': [Rules.python_wrap_html, Rules.python_concat_html, Rules.python_wrap_html]
+        },
+        {
+            'expression':
+                textwrap.dedent("""
```

Comment:
I expected to see more new test cases.  Maybe I don't understand what the new code enables?